lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -150,6 +150,20 @@ def which(cmd):
     LOGGER.warning("Could not find command '%s' (with permissions to read/execute it) in $PATH (%s)", cmd, paths)
     return None
 
+def check_output(time_passed, output_timeout, seen_output):
+    if time_passed > output_timeout and not seen_output:
+        warning_msg = '\n'.join([
+            "WARNING: mympirun has been running for %s seconds without seeing any output." % time_passed,
+            "This may mean that your program is hanging, please check and make sure that is not the case!",
+            '',
+            "If this warning is printed too soon and the program is doing useful work without producing any output,",
+            "you can increase the timeout threshold via --output-check-timeout (current setting: %s seconds)" % output_timeout,
+            ])
+        LOGGER.warn(warning_msg + '\n')
+        seen_output = True
+
+    return seen_output
```

Comment:
why not return the actual message itself, or `None` if no message should be printed?